data/states.js

```diff
@@ -0,0 +1,282 @@
+var stateData = [
```

Comment:
I could also store the FIPS on the event, which would eliminate the lookup on this end